src/components/Search/beta/SearchDropdown.tsx

```diff
@@ -20,12 +20,46 @@ import {
   DropDownMenuRaw,
   DropDownItemRaw,
 } from '../../../styles';
+import type { Repository } from '../../../types/state';
 
-function SearchDropdown(props: SearchDropdownProps) {
-  const { repository, view, selectLabel, tooltipText, mode } = props;
+function SearchDropdown({
+  repository,
+  view,
+  selectLabel,
+  tooltipText,
+  mode,
+  base,
+}: SearchDropdownProps) {
   const size = view == 'compare-results' ? 'small' : undefined;
   const { handleChangeDropdown } = useHandleChangeDropdown();
 
+  //searchType is to distinguish between base and new search dropdowns for handleChangeDropdown hook
+  const [repoSelect, setRepoSelect] = useState({
+    baseRepository: 'try',
+    newRepository: 'try',
+    searchType: base,
+  });
+
+  useEffect(() => {
+    void handleChangeDropdown(repoSelect);
+  }, [repoSelect]);
```

Comment:
It's not clear to me why you don't call it directly in `handleRepoSelect` below.